package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@azure/service-bus",
+  "version": "0.1.0",
+  "description": "Azure ServiceBus SDK for JS",
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "main": "./dist/lib/index.js",
+  "types": "./typings/lib/index.d.ts",
+  "dependencies": {
+    "@azure/amqp-common": "azure/amqp-common-js",
```

Comment:
is this intentional?

Reply:
I have to make changes to the EventProcessorHost package, before publishing the 0.1.4 version of amqp-common. Hence, have taken a dependency on the git repo. Will update this after making changes to EPH sdk.